src/styles/index.scss

```diff
@@ -1,3 +1,58 @@
+@import './utils/variables';
+@import 'block/header';
+@import 'block/pruduct-card';
+@import 'block/starts';
```

Comment:
Typo in the file name import: 'starts' should be 'stars' if it is meant to represent a rating system.